bundles/aws/build.gradle.kts

```diff
@@ -40,15 +43,12 @@ dependencies {
     exclude("*")
   }
 
-  implementation(libs.aws.iam)
-  implementation(libs.aws.policy)
```

Comment:
s3 token provider relies on these packages.

Reply:
According to my understanding, there is no such assumption.  the bundle jars is used to provide the dependencies jars. cc @yuqi1129

Reply:
For the user cases which doesn't need hadoop and aws jars, another solution is we did't provide any bundle jars.  I guess user may use some code in bundle, we could move this part out of bundle.

Reply:
> Since lacking so many dependencies, how can we assure that it's easy for users to use the jar. 

It is because we assume that the user environment already has access to cloud storage, so these users find it easy to use. If not, they should use `gravitino-aws-bundle.jar`.

> is it reasonable to exclude dependencies like libs.aws.iam, libs.aws.policy?

`libs.aws.iam` and `libs.aws.policy` are also a part of aws dependency, so I think it's reasonable to exclude them from `gravitino-aws.jar`

Reply:
> I also need to put `libs.aws.policy` and `libs.aws.iam` in the class path to make it runs

Are you sure? If it's just simple file access, I think there's no need to introduce these two dependencies. (the test `TestCredentialProvider` in this module runs well and we only use `testImplementation(libs.aws.s3)`)

Reply:
@FANNG1